docs/6.0. Upgrade Path.md

```diff
@@ -8,18 +8,55 @@ API versioning is specified in the [APIs](2.0.%20APIs.md) documentation, with pr
 
 Registrations with a Registration API must only proceed if the Node API version implemented exactly matches the API version used by the Registration API.
 
 ## Requirements for Registries (Registration and Query APIs)
 
 Implementers of the Registration and Query API must support at least one API version. It is however strongly recommended that Registration and Query APIs fully support at least two and preferably more consecutive API versions (if released). In doing so, facilities which include a large number of Nodes may stagger their equipment upgrades whilst maintaining compatibility with a single registry.
 
-When supporting multiple API versions, Query APIs must provide translations of resources for backwards compatibility. For example, if the registry contains a mixture of v1.0 and v1.1 resources, a v1.0 Query API must provide a response containing all of these resources by removing keys from v1.1 resources which are not present in v1.0.
+When supporting multiple API versions, Query APIs must provide translations of resources for backwards compatibility (see [Version Translations](#version-translations)). For example, if the registry contains a mixture of v1.0 and v1.1 resources, a v1.0 Query API must provide a response containing all of these resources by removing keys from v1.1 resources which are not present in v1.0. Translations must not be provided for '/subscriptions' endpoints, which only expose subscriptions matching the endpoint's API version.
 
 Query APIs are not required to provide for forwards compatibility as it may be impossible to generate data for new attributes in schemas. Query APIs should however allow clients to request older data than the requested minor API version by using the ?query.downgrade=<version> query parameter (see Query API documentation for examples).
 
+### Version Translations
+
+When conforming a resource to an earlier API version, the following keys MUST be removed. These are documented here for clarity, but the same data could be derived from analysis of the schema changes between API versions.
+
+Note that removal of these keys does not guarantee conformance to the schema of the earlier API version due to the addition of new 'enum' values and similar features in more recent API versions. As such, Query APIs SHOULD NOT validate translated resources against schemas. Nodes MUST ensure that any translated resources meet the requirements of earlier schemas if they could be required to communicate with an earlier versioned Registration API.
+
```

Comment:
Not quite sure I understand the final sentence. Does this mean that a Node that supports v1.2 and v1.1 but finds itself communicating with a v1.1 Registration API must make registration requests which are valid according to the v1.1 schemas?

Reply:
Yes, that was the intention.